MDC-101: Material Components (MDC) Basics (Flutter)

Attempt to download and build MDC 101 Starter Project. 2 problems ( 1) is probably not important, 2) is a show stopper.

1) Directions (item 4) says
 ****4. In the project panel on the left, delete the testing file ../test/widget_test.dart**** 
 file is not found

2) Project does not build. Cannot find **mips64el-linux-android** in the toolchain
==============================================================

When I follow instructions on https://codelabs.developers.google.com/codelabs/mdc-101-flutter/index.html?index=..%2F..index#2
and use the “Download Starter Project” option, Android Studio shows me this:
![fig2](https://user-images.githubusercontent.com/9920713/49909717-ee9fba80-fe34-11e8-99a0-a4bc8f41bbf6.png)
which indicates a problem:
Dart support is not enabled…
I click on ‘Enable Dart support’ that seems to fix the problem, but now I get:
![fig1](https://user-images.githubusercontent.com/9920713/49909776-1f7fef80-fe35-11e8-906c-9793c3fc60d8.png)
So ‘Packages get’ has not been run. I run it by clicking on ‘Get dependencies’ and I get this:
![fig4](https://user-images.githubusercontent.com/9920713/49910018-03c91900-fe36-11e8-9029-730f64ca55d9.png)

So it looks good…...Let’s delete that test file
4. In the project panel on the left, delete the testing file ../test/widget_test.dart 

….Can’t find widget_test.* 


Attempt to build project:
Get the following error:

Launching lib\main.dart on Android SDK built for x86 in debug mode...
Initializing gradle...
Resolving dependencies...
* Error running Gradle:
ProcessException: Process "C:\etc\prj\material-components-flutter-codelabs-101-starter\material-components-flutter-codelabs-101-starter\mdc_100_series\android\gradlew.bat" exited abnormally:
Starting a Gradle Daemon (subsequent builds will be faster)


FAILURE: Build failed with an exception.

* Where:
Build file 'C:\etc\prj\material-components-flutter-codelabs-101-starter\material-components-flutter-codelabs-101-starter\mdc_100_series\android\build.gradle' line: 24

* What went wrong:
A problem occurred evaluating root project 'android'.
> A problem occurred configuring project ':app'.
   > No toolchains found in the NDK toolchains folder for ABI with prefix: mips64el-linux-android

--- a/mdc_100_series/android/build.gradle
+++ b/mdc_100_series/android/build.gradle
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.2.1'
     }
 }
 

--- a/mdc_100_series/android/gradle/wrapper/gradle-wrapper.properties
+++ b/mdc_100_series/android/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip

